biolink-model.yaml

```diff
@@ -11447,6 +11474,15 @@ enums:
           A PubMed text-mining score from Jensen Lab less than 5, greater than or equal TO 3 Gene RIFs, or
           less than or equal to 50 Antibodies available according to http://antibodypedia.com.
 
+  genetic_inheritance_enum:
+    reachable_from:
+      source_ontology: bioregistry:geno
+      source_nodes:
+        - GENO:0000141  ## inheritance pattern
```

Comment:
I notice that the **GENO** ontology is assumed here. As it happens, in Monarch, we are getting our inheritance data via HPOA (actually, `HPO_ID` values in the **phenotype.hpoa** input data file).  How do we wish to handle this dichotomy of ontology terms in the Biolink Model (and Monarch, for that matter). Is it necessary to remap the HPO terms (say, in the Monarch Ingest) into GENO enum values, before the resulting graph data (i.e. KGX output) is deemed fully Biolink Model compliant, or would we somehow allow both ontology subtrees into the enum. 

This strikes me as a general challenge of enum's for specific (category/node property/association edge) slot values which we choose to model as enum's. I can imagine, in many cases, that the mappings may often be one-to-one, but how do we express this in the Biolink Model? Maybe SSSOM files associated with model enums, LOL?